source/NVDAObjects/IAccessible/ia2TextMozilla.py

```diff
@@ -307,7 +307,7 @@ def _getText(self, withFields, formatConfig=None):
 		while fields[-1] is not None:
 			# The end hasn't yet been reached, which means it isn't a descendant of obj.
 			# Therefore, continue from where obj was embedded.
-			if withFields:
+			if withFields and controlStack:

```

Comment:
Ug. Tried to do this as a multi-line suggestion but can't figure out how without a mouse.

```suggestion
			if withFields:
```